pkg/controller/workspacerouting/solvers/solver.go

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package solvers
+
+import (
+	"github.com/che-incubator/che-workspace-operator/pkg/apis/workspace/v1alpha1"
+	v12 "github.com/openshift/api/route/v1"
```

Comment:
openshift api is so far from v12 ;)
```suggestion
	routev1 "github.com/openshift/api/route/v1"
```

Reply:
Yeah, this is a goland problem, I thought I caught all instances of it doing this :angry: 

If you have an import alias conflict it just throws a `2` on the end silently.